build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
+
+RUN yum update -y && yum install -y git curl bzip2 wget flex fontconfig libX11 zlib-devel
+RUN yum install -y centos-release-scl && yum install -y devtoolset-8 texinfo
+SHELL [ "/usr/bin/scl", "enable", "devtoolset-8" ]
+
+#
+# Centos 8
+#
+
+#FROM centos:centos8
+
+#RUN yum update -y && yum install -y  git curl bzip2 wget flex fontconfig libX11 zlib-devel
+#RUN yum install -y --enablerepo=powertools gcc texinfo make
```

Comment:
Coherently with the previous comment, split L21 to a stage.

Reply:
Done and left commented the centos 8 part (no clear to me how to call right now).

Reply:
See my comment above, about installing GNAT before the version-specific stages.

Then, you need to add the version to the stage names:

```
FROM common AS gcc-7
...

FROM common AS gcc-8
...
```

Last, in the `run.sh` script, modify the `--target="$tgt"` argument to `--target="${tgt}${VERSION}"`.